firefox_media_tests/playback/netflix/test_basic_playback.py

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from media_test_harness.testcase import MediaTestCase
+from firefox_media_tests.playback.video_playback_tests_mixin import (
+    VideoPlaybackTestsMixin
+)
+
+
+class TestBasicNetflixPlayback(MediaTestCase,
+                               VideoPlaybackTestsMixin):
+
+    def setUp(self):
+        super(TestBasicNetflixPlayback, self).setUp()
+        self.set_eme_prefs()
+        assert(self.check_eme_prefs())
+
+    def set_eme_prefs(self):
+        with self.marionette.using_context('chrome'):

```

Comment:
Suggestion: add a comment that refers to the bug where cpearce suggestion to try this pref